ipapython/ipaldap.py

```diff
@@ -1639,48 +1636,25 @@ def __init__(self, host='', port=389, cacert=None, debug=None, ldapi=False,
     def __str__(self):
         return self.host + ":" + str(self.port)
 
-    def __wait_for_connection(self, timeout):
```

Comment:
LGTM
https://fedorahosted.org/freeipa/ticket/2175

Upgrade is using ldapi now.

However I'm curious if there is a time delay between DS is listening on ldapi socket and when DS is listening on 389 port. But in our case we plan to use just ldapi, so it should be ok.